src/main/java/core/basesyntax/service/ColorSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int randomIndex = ThreadLocalRandom.current().nextInt(Color.values().length);
+        return Color.values()[randomIndex].getColorName();
+    }
+
+    private enum Color {
+        RED, GREEN, BLACK, BLUE, WHITE;
+
+        public String getColorName() {
+            return this.name().charAt(0) + this.name().substring(1).toLowerCase();
+        }
```

Comment:
Don't use `toString()` or methods that modify the `toString()` representation for getting the String representation of enum constants. Instead, use the `name()` method. In this case, you can remove `getColorName()` and directly use `name()` in line 8.